sdcm/nemesis.py

```diff
@@ -3885,6 +3912,20 @@ def disrupt_run_unique_sequence(self):
         self.disrupt_grow_shrink_cluster()
         InfoEvent(message="Finished grow_shrink disruption").publish()
 
+    def disrupt_run_unique_delete_range_sequence(self):
+        InfoEvent(message='Start - delete range').publish()
+        run_func = random.choice([self.disrupt_delete_10_full_partitions,  self.disrupt_delete_overlapping_row_ranges,
+                                  self.disrupt_delete_by_rows_range])
+        run_func()
+        InfoEvent(message='Finish - delete range').publish()
+
+        # self.target_node.run_nodetool(sub_cmd="flush", args="scylla_bench")
+
+        InfoEvent(message='Start - destroy data and repair').publish()
+        self._destroy_data_and_restart_scylla(keyspace_for_destroy="scylla_bench", destroy_all_files=True)
```

Comment:
Can we just select all test keyspaces ?

Reply:
This is the goal here - destroy data in the table with large partitions and MV